docs/readthedocs/source/doc/LLM/Quickstart/install_linux_driver.md

```diff
@@ -0,0 +1,82 @@
+# Install GPU Driver on Linux
+
+This guide demonstrates how to install Intel GPU driver on linux with kernel version 5.19/
+
+It applies to Intel Arc Series GPU.
+
+* install arc driver
+    ```bash
+    sudo apt-get install -y gpg-agent wget
+    wget -qO - https://repositories.intel.com/gpu/intel-graphics.key | \
+    sudo gpg --dearmor --output /usr/share/keyrings/intel-graphics.gpg
+    echo "deb [arch=amd64,i386 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/gpu/ubuntu jammy client" | \
+    sudo tee /etc/apt/sources.list.d/intel-gpu-jammy.list
+    ```
+
+* downgrade kernel
+    ```bash
+    sudo apt-get update && sudo apt-get install  -y --install-suggests  linux-image-5.19.0-41-generic
+
+    sudo sed -i "s/GRUB_DEFAULT=.*/GRUB_DEFAULT=\"1> $(echo $(($(awk -F\' '/menuentry / {print $2}' /boot/grub/grub.cfg \
+    | grep -no '5.19.0-41' | sed 's/:/\n/g' | head -n 1)-2)))\"/" /etc/default/grub
+
+    sudo  update-grub
+
+    sudo reboot
+    # As 5.19's kernel doesn't has any arc graphic driver. The machine may not start the desktop correctly, but we can use the ssh to login. 
+    # Or you can select 5.19's recovery mode in the grub, then choose resume to resume the normal boot directly.
+
+    # remove latest kernel
+
+    sudo apt purge linux-image-6.2.0-*
```

Comment:
We don't need to remove 6.2's kernel now. 6.2 is supported.

Reply:
> We don't need to remove 6.2's kernel now. 6.2 is supported.

Fixed